Spigot-API-Patches/0218-Add-setMaxPlayers-API.patch

```diff
@@ -0,0 +1,48 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mariell Hoversholm <proximyst@proximyst.com>
+Date: Sat, 22 Aug 2020 23:59:25 +0200
+Subject: [PATCH] Add #setMaxPlayers API
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index 62cc1c74c11f56dcbd1e24e9c5478497742e6351..7fd92462189be35808af67be1740345f186ce555 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -171,6 +171,17 @@ public final class Bukkit {
+         return server.getMaxPlayers();
+     }
+ 
++    // Paper start
++    /**
++     * Set the maximum amount of players which can login to this server.
++     *
++     * @param the amount of players this server allows
++     */
++    public static void setMaxPlayers(int maxPlayers) {
++        server.setMaxPlayers(maxPlayers);
++    }
++    // Paper end
++
+     /**
+      * Get the game port that the server runs on.
+      *
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index 6e01bf2d52e8bb6de7395f50c12f16c64aef72ae..bacb3ee9abe5fe26e02caa7f54fc4e6d5dd605fe 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -144,6 +144,15 @@ public interface Server extends PluginMessageRecipient {
+      */
+     public int getMaxPlayers();
+ 
++    // Paper start
++    /**
++     * Set the maximum amount of players which can login to this server.
++     *
++     * @param the amount of players this server allows
```

Comment:
`maxPlayers` missing in `@param` (makes Jenkins fail builds when creating Javadocs)